pulsar-function-go/go.mod

```diff
@@ -3,15 +3,16 @@ module github.com/apache/pulsar/pulsar-function-go
 go 1.13
 
 require (
-	github.com/apache/pulsar-client-go v0.7.0
-	github.com/golang/protobuf v1.4.3
-	github.com/prometheus/client_golang v1.7.1
+	github.com/apache/pulsar-client-go v0.8.0
+	github.com/dgrijalva/jwt-go v3.2.0+incompatible // indirect
+	github.com/golang/protobuf v1.5.2
+	github.com/prometheus/client_golang v1.11.0
```

Comment:
1.11.0 is flagged with a critical CVE, could you upgrade it to 1.11.1 ?

Reply:
Good ideas, will upgrade go sdk version to 081